### DISCUSSION TITLE
Having a “replace prettier” checklist

### DISCUSSION THREAD

**Discussion by AbdealiLoKo** [biomejs/biome#154] [General]
I have recently moved from flake8 to ruff and the ruff team gave a very nice and clean Checklist that allowed me to identify exactly what flake8 can do vs ruff can do.

is there some clear docs for what biome can do and what prettier can do ?

Something similar to https://github.com/biomejs/biome/discussions/3

**Comment by ematipico** [comment]
I believe this is a good material for a guide of our website.

I still haven't had the chance to migrate a prettier project to biome (I'm in the process though).

If you or another member of the community wants to pick it up, by all means, feel free to do so! I will help by answering all your questions
